integration/test/Test/Spar.hs

```diff
@@ -472,21 +472,40 @@ testSparScimUpdateUserGroup = do
               .= [ object
                      [ "value" .= charlieId,
                        "type" .= "User",
-                       "$ref" .= ("http://example.com:8088/scim/v2/Users/" <> charlieId)
+                       "$ref" .= ("http://localhost:8088/scim/v2/Users/" <> charlieId)
                      ],
                    object
                      [ "value" .= dianaId,
                        "type" .= "User",
-                       "$ref" .= ("http://example.com:8088/scim/v2/Users/" <> dianaId)
+                       "$ref" .= ("http://localhost:8088/scim/v2/Users/" <> dianaId)
                      ]
                  ]
           ]
-  updateScimUserGroup OwnDomain tok gid scimUserGroupUpdated `bindResponse` \resp -> do
-    resp.status `shouldMatchInt` 200
-  getScimUserGroup OwnDomain tok gid `bindResponse` \resp -> do
-    resp.json %. "displayName" `shouldMatch` "My even funkier group"
-    memberValues <- (resp.json %. "members") >>= asListOf (\m -> m %. "value" >>= asString)
-    memberValues `shouldMatchSet` [charlieId, dianaId]
+  -- run this twice to check for idempotency
+  replicateM_ 2 $ do
+    updateScimUserGroup OwnDomain tok gid scimUserGroupUpdated `bindResponse` \resp -> do
+      resp.status `shouldMatchInt` 200
+      let expectedMembers =
+            [ object
+                [ "value" .= charlieId,
+                  "type" .= "User",
+                  "$ref" .= ("http://localhost:8088/scim/v2/Users/" <> charlieId)
+                ],
+              object
+                [ "value" .= dianaId,
+                  "type" .= "User",
+                  "$ref" .= ("http://localhost:8088/scim/v2/Users/" <> dianaId)
+                ]
+            ]
+      resp.json %. "displayName" `shouldMatch` "My even funkier group"
+      asList (resp.json %. "members") `shouldMatchSet` expectedMembers
+      resp.json %. "id" `shouldMatch` gid
+      resp.json %. "schemas" `shouldMatch` (toJSON ["urn:ietf:params:scim:schemas:core:2.0:Group"])
+    -- TODO: check meta, too?
```

Comment:
i've checked it for Get in the unit tests, but not for update (yet).  but yes, we should test it either here or there.